.editor_template/README.md

```diff
@@ -0,0 +1,9 @@
+## Editor-name configuration
+
+### Installation
+
+Instruction with steps
+- how
```

Comment:
@kotylamichal maybe lets add here three sections. How to install given editor, how to install configuration, which extensions are useful and how we can install them.